src/wolfgame.py

```diff
@@ -6161,16 +6160,15 @@ def pony(cli, nick, chan, rest):
     elif rnd < 0.95:
         pony = messages["pony_choices"][2].format(nick=nick)
     else:
-        reply(cli, nick, chan, messages["pony_fly"])
+        wrapper.send(messages["pony_fly"])
         return
-    cmsg = messages["pony_land"].format(pony)
-    reply(cli, nick, chan, cmsg)
+    wrapper.send(messages["pony_land"].format(pony))
 
-@cmd("cat", pm=True)
-def cat(cli, nick, chan, rest):
+@command("cat", pm=True)
+def cat(var, wrapper, message):
     """Toss a cat into the air and see what happens!"""
-    reply(cli, nick, chan, messages["cat_toss"].format(nick))
-    reply(cli, nick, chan, messages["cat_land"])
+    wrapper.send(messages["cat_toss"].format(wrapper.source.nick))
+    wrapper.send(messages["cat_land"])
```

Comment:
Same here too, use `wrapper.source` directly. Also, the two lines can be merged into one by adding the second line into the first call with a comma, and specifying `sep="\n"` in the call.